src/annotator/integrations/vitalsource.ts

```diff
@@ -484,49 +465,15 @@ export class VitalSourceContentIntegration
   }
 
   async uri() {
-    if (this._bookIsSingleDocument()) {
-      const bookInfo = this._bookElement.getBookInfo();
-      const bookId = bookInfo.isbn;
-      return `https://bookshelf.vitalsource.com/reader/books/${bookId}`;
+    const bookInfo = this._bookElement.getBookInfo();
+    const bookId = bookInfo.isbn;
+    if (!bookId) {
+      throw new Error('Unable to get book ID from VitalSource');
     }
-
-    // An example of a typical URL for the chapter content in the Bookshelf reader is:
-    //
-    // https://jigsaw.vitalsource.com/books/9781848317703/epub/OPS/xhtml/chapter_001.html#cfi=/6/10%5B;vnd.vst.idref=chap001%5D!/4
-    //
-    // Where "9781848317703" is the VitalSource book ID ("vbid"), "chapter_001.html"
-    // is the location of the HTML page for the current chapter within the book
-    // and the `#cfi` fragment identifies the scroll location.
-    //
-    // Note that this URL is typically different than what is displayed in the
-    // iframe's `src` attribute.
-
-    // Strip off search parameters and fragments.
```

Comment:
None of this commenting is relevant anymore? Just double-checking.

Reply:
Indeed, because we're not using this URL as the annotation URL any more.